src/corecel/grid/FindInterp.hh

```diff
@@ -13,8 +13,6 @@
 
 namespace celeritas
 {
-namespace detail
-{
```

Comment:
The doxygen `\file` needs to be updated as well.

Reply:
Thanks!  Please do not merge this yet as I have another modification.